lib/rubygems/test_case.rb

```diff
@@ -24,7 +24,9 @@
   gem 'rdoc'
   gem 'json'
 end
+Gem.ui
```

Comment:
it would be super great to have a comment indicating what side effect this causes that is required—without a comment like that, I am likely to delete it later :P